### PR TITLE
[Fix] Fix bug of human skeleton grouping for params.ignore_too_much

### PR DESCRIPTION
**Pull request by daixinghome**
## Motivation

Fix: https://github.com/open-mmlab/mmpose/issues/1336

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by jin-s13** [comment]
Thank you for your contribution! @daixinghome 
It seems that the lint check fails. Would you mind following the https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md to setup the pre-commit?

**Comment by daixinghome** [comment]
> Thank you for your contribution! @daixinghome It seems that the lint check fails. Would you mind following the https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md to setup the pre-commit?

I'm sorry for the mistakes. Now it's ok with my pre-commit's checking.

**Comment by daixinghome** [comment]
Well, it seems that there is no testcase for ignore_too_much parameter in master version. I add the testing now.

File: mmpose/core/post_processing/group.py [hunk 1]
```diff
@@ -63,8 +63,8 @@ def _match_by_tag(inp, params):
         tags = tag_k[idx]
         joints = np.concatenate((loc_k[idx], val_k[idx, :, None], tags), 1)
         mask = joints[:, 2] > params.detection_threshold
-        tags = tags[mask]
-        joints = joints[mask]
+        tags = tags[mask]  # shape: [M, L]
```
Comment (jin-s13) [inline mmpose/core/post_processing/group.py:66]: Please delete the code comment or make it a new line on the top.

**Review by jin-s13 — APPROVED** [review]
This PR looks good to me in general. And I have tested the accuracy, which is good also.
One minor comment is to unify the style of code commenting.

**Comment by OpenMMLab-Coordinator** [comment]
daixinghome，您好！您在MMPose项目中给我们提的PR非常重要，感谢您付出私人时间帮助改进开源项目，相信很多开发者会从你的PR中受益。
我们非常期待与您继续合作，OpenMMLab专门成立了贡献者组织MMSIG，为贡献者们提供开源证书、荣誉体系和专享好礼，可通过添加微信：openmmlabwx 联系我们（请备注mmsig+GitHub id），由衷希望您能加入！


Hi @daixinghome ！First of all, we want to express our gratitude for your significant PR in the MMPose project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**